.github/workflows/integration.yml

```diff
@@ -25,19 +25,10 @@ jobs:
     - name: Run all tests
       run: cargo test --all
```

Comment:
Since you're already making improvements:

Can you modify this to: `cargo test --all-features`?
More UTs will be run that way.

Reply:
Sure, please let me update it